plugins/module_utils/aci.py

```diff
@@ -1429,10 +1433,25 @@ def get_existing(self):
         that this method can be used to supply the existing configuration when using the get_diff method. The response, status,
         and existing configuration will be added to the self.result dictionary.
         """
-        uri = self.url + self.filter_string
+        if self.suppress_look_back:
+            self.existing = []
+            return
 
+        uri = self.url + self.filter_string
         self.api_call("GET", uri, data=None, return_response=False)
 
+    def __get_existing_validation(self, changed):
+        if self.suppress_verification:
```

Comment:
What's the expected behaviour when `suppress_look_back` is set to True but `suppress_verification` is set to False here? or does it not matter because "current_verified" is unrelated to `suppress_look_back`?

Reply:
Its sort of unrelated because one stops the GET before the POST and the other stops the GET after the POST.
However, in this case when `suppress_previous` is true and `suppress_verification` is false, a POST will be sent first and a verifying GET will also be sent after. The expected behaviour would be `previous` is empty and the `current` is the result of the POST based on the GET.
With both flags on modules will just send POSTs. Resulting in `previous` being empty and `current` being the same as `proposed`.